app/gps/Export/ExportCodeInput.js

```diff
@@ -198,6 +205,16 @@ export const ExportSelectHA = ({ route, navigation }) => {
               label={t('common.next')}
               onPress={validateCode}
             />
+            {/* Bypass API to allow easy testing of screens */}
+            <FeatureFlag flag={FeatureFlagOption.BYPASS_EXPORT_API}>
```

Comment:
Just a thought, since this will bypass the upload step anyway, should we just always validate the code instead of having a second button?

Reply:
💯 That makes a lot more sense given how the other screens bypass on the same buttons